.travis.yml

```diff
@@ -6,16 +6,19 @@ php:
 
 sudo: false
 
+env:
+  global:
+    - TZ=Europe/Paris
```

Comment:
See https://docs.travis-ci.com/user/environment-variables/.

Reply:
Also see this in effect here: https://travis-ci.org/phpbench/phpbench/jobs/285491568#L429-L430.

Reply:
Can't hurt, but what is the advantage?

Reply:
In my opinion, the advantage is that it's more obviously stated that we define an environment variable used in the builds.

What do you think?

Reply:
I guess I'd prefer to leave it out unless the TZ is having a beneficial effect on the build, looking at the Symfony `.travis.yml` they don't seem to include this

Reply:
No problem, let me sort this out later.

Reply:
Cool, thanks for the improvements so far :)